app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +92,63 @@ class Identification extends React.Component {
       });
   }
 
+  getLanguageOptions(languageType) {
+    const langOptions = supportedLanguages.languages.map(function(lang) {
+      let fullySupported = lang.supported.sms && lang.supported.email && lang.supported.phone;
+      let langLabel = fullySupported ? lang.name : lang.name + '*';
+      return { value: lang.name, label: langLabel, type: languageType };
+    });
+    return langOptions;
+  }
+
+  getLanguageValue(language, languageType) {
+    return this.getLanguageOptions(languageType).filter(lang => lang.value === language);
```

Comment:
This is looking for a single selected value of all options, so I think `Array.find() `would be faster than `Array.filter()` here.